src/WebJobs.Script/Host/ScriptHostIdProvider.cs

```diff
@@ -50,7 +50,7 @@ internal static HostIdResult GetDefaultHostId(IEnvironment environment, ScriptAp
         {
             HostIdResult result = new HostIdResult();
 
-            if (environment.IsFlexConsumptionSku())
+            if (environment.IsFlexConsumptionSku() || environment.IsConsumptionV1RunningOnLegion())
```

Comment:
Why are you making this change? We don't intend to change host behaviors for Cv1 when running on Legion. It's just a migration.